.github/workflows/playwright.yml

```diff
@@ -0,0 +1,29 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
```

Comment:
Do we want to be explicit about which node.js versions this runs on? What versions do we officially support with Tabulator -- I'd make this list the same.